start_node.sh

```diff
@@ -0,0 +1,45 @@
+KEY="mykey"
```

Comment:
I think this file came from local testing? Or is this intentionally added & also a scope of this PR?

Reply:
ohh it's a mistake, sorry

Reply:
I've deleted it

Reply:
nice thanks!